ReadMe.md

```diff
@@ -0,0 +1,12 @@
+## Get Binary Dependencies
+
+In the `build` directory, run
+
+`buildupdate.windows.sh`
```

Comment:
Do you need to specify project install requirements (e.g. VS 2015, git for windows--so that you have bash)?